charts/oam-core-runtime/.gitignore

```diff
@@ -0,0 +1,2 @@
+values.yaml
```

Comment:
I didn't get why the `value.yaml` file is renamed and also ignored here. What is that `values.yaml-e` file?

I also don't know why it is renamed.

Reply:
`values.yaml` is now `values.yaml.tmpl`. This is just how the `build` machinery works. It uses `values.yaml.tmpl` to generate `values.yaml` during the build. This allows you to do something like this https://github.com/crossplane/oam-kubernetes-runtime/pull/90/files#diff-b67911656ef5d18c4ae36cb6741b7965R28